test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -194,6 +194,13 @@ context('action sidebar', function() {
 
     cy
       .server()
+      .routeGroups(fx => {
+        fx.data[2].relationships.clinicians.data[1] = { id: '22222', type: 'clinicians' };
+        fx.included[0].id = '22222';
```

Comment:
probably should set the `type` here too to future proof any other includes being added to this groups call.  Might be safer to just do `fx.included[0] = { ... }` or to `_.find(fx.included, { type: 'clinicians' })`